tuplex/core/include/physical/TransformTask.h

```diff
@@ -225,6 +225,14 @@ namespace tuplex {
         */
         std::unordered_map<std::tuple<int64_t, ExceptionCode>, size_t> exceptionCounts() const { return _exceptionCounts; }
 
+        std::string partitionId() {
```

Comment:
what does this function do? It seems to give the UUID of the first partition. shouldn't it be called then firstPartitionID()?

Pls add doxygen for this function